src/propTypeExtras.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import PropTypes from 'prop-types'
+type ValidatorType = (props: Object, propName: string, componentName: string, ...rest: Array<any>) => Error | null
```

Comment:
Yep. Type definitions were wrong here which was brought to the surface by new Flow. This fix is grouped with other related changes in "Update Flow configuration " commit.

Reply:
Were ok in old flow ... but new flow did make a bunch of breaking changes (or at least did last time I looked at upgrading it)

Reply:
If we had correct type definition it would work:) New flow is more strict (better?) at type checking and started reporting errors that previously were hidden.